src/renderers/js-experimental/fragments/programInstructions.ts

```diff
@@ -29,6 +30,7 @@ export function getProgramInstructionsFragment(
     [
       getProgramInstructionsEnumFragment(scopeWithInstructions),
       getProgramInstructionsIdentifierFunctionFragment(scopeWithInstructions),
+      getProgramUnionParsedInstructionTypeFragment(scopeWithInstructions),
```

Comment:
nit: I've tried to follow a convention that groups function and type names together by starting with the same prefix. I also try and make the last word describe what it is generating (e.g. `Function`, `Enum` or `Type`). So here it would be more like:

```suggestion
      getProgramInstructionsParsedUnionTypeFragment(scopeWithInstructions),
```